projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-avatar.spec.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable rxjs/no-topromise */
```

Comment:
I think that we should disable this rule for all `**/schematics/**/*.spec.ts` files.

Now every test file for schematics contains this line.

Reply:
@splincode what do you think ?

Could we update this file ?
https://github.com/taiga-family/linters/blob/main/projects/eslint-plugin-experience/configs/angular/rxjs/index.js

Reply:
Every test file for schematics contains:

```ts
const tree = await runner
    .runSchematicAsync(...)
    .toPromise();
```

Reply:
I will do it right now